app/src/main/java/mozilla/org/webmaker/WebmakerApplication.java

```diff
@@ -30,10 +30,10 @@ public void onCreate() {
         Router.sharedRouter().setContext(getApplicationContext());
         Router.sharedRouter().map("/main", MainActivity.class);
         Router.sharedRouter().map("/main/:tab", MainActivity.class);
-        Router.sharedRouter().map("/map/:projectId", Project.class);
-        Router.sharedRouter().map("/projects/:projectId", Page.class);
-        Router.sharedRouter().map("/elements/:projectId/:elementId/:type", Element.class);
-        Router.sharedRouter().map("/projects/:projectId/elements/:elementId/:attributeId", Tinker.class);
+        Router.sharedRouter().map("/projects/:project", Project.class);
+        Router.sharedRouter().map("/projects/:project/pages/:page", Page.class);
```

Comment:
I'd recommend making these match the thing they're loading instead of wildly overloading the /projects route and using a mixture of :params and virtual directory labels to hope things work out.